extensions/ql-vscode/src/vscode-tests/utils/bundled-pack-helpers.ts

```diff
@@ -0,0 +1,76 @@
+import { Readable } from 'stream';
+import * as tar from 'tar-stream';
+import { Headers } from 'tar-stream';
+import { pipeline } from 'stream/promises';
+import { createGunzip } from 'zlib';
+
+export interface QueryPackFS {
+  fileExists: (name: string) => boolean;
+  fileContents: (name: string) => Buffer;
+  directoryContents: (name: string) => string[];
+}
+
+export const readBundledPack = async (base64Pack: string): Promise<QueryPackFS> => {
```

Comment:
(totally minor and optional) I think we've generally followed the `export function()` style so might be good to stick to that.